resources/public/api/conf/2.0/schemas/foreign_property_accounting_adjustments_submit_request.json

```diff
@@ -5,9 +5,15 @@
   "type": "object",
   "properties": {
     "foreignProperty": {
-      "type": "object",
+      "type": "array",
       "description": "Object holding foreign property adjustments.",
+      "items": {
+        "type": "object",
       "properties": {
+        "countryCode": {
+          "description": "A three-letter code that represents a country name.\n\nThis must be an ISO 3166-1 Alpha-3 value.\n\nFor example: FRA",
+          "type": "string"
+        },
         "income": {
```

Comment:
foreignTaxTakenOff will be removed from what is submited in income. Alos can it all be formated please